src/components/Settings/actions/appearance.js

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef } from 'react'
+
+import flag from 'cozy-flags'
+import ActionsMenuItem from 'cozy-ui/transpiled/react/ActionsMenu/ActionsMenuItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import PaletteIcon from 'cozy-ui/transpiled/react/Icons/Palette'
+
+import { getSettingsLink } from 'components/Settings/helper'
+
+const Component = forwardRef(({ action, ...props }, ref) => {
+  return (
+    <ActionsMenuItem {...props} ref={ref}>
+      <ListItemIcon>
+        <Icon icon={action.icon} />
+      </ListItemIcon>
+      <ListItemText primary={action.label} />
+    </ActionsMenuItem>
+  )
+})
```

Comment:
A other point for cozy-ui, maybe could we expose this component directly from cozy-ui as it will be the same in many place